src/test/java/bio/terra/endpoints/JavaHTTPRequestTests.java

```diff
@@ -0,0 +1,125 @@
+package bio.terra.endpoints;
```

Comment:
These don't seem like CLI tests. What did you have in mind here?

Reply:
Right, these are not CLI tests. They are intended to help with debugging CLI failures by checking that the Data Repo instance is responding to basic unauthenticated and authenticated HTTP requests. This was helpful for me as I was debugging to see whether the problem was with the CLI or my test versus with the instance I was connecting to or the credentials I was using.

I put this comment at the top of the file also, by way of explanation.